src/Stratis.StratisDnsD/Program.cs

```diff
@@ -56,8 +56,8 @@ public static void Main(string[] args)
                     // Build the Dns full node.
                     node = new FullNodeBuilder()
                         .UseNodeSettings(nodeSettings)
-                        .UsePosConsensus()
                         .UseBlockStore()
+                        .UsePosConsensus()
```

Comment:
why the order switch?

Reply:
BlockStore.cs: 114
// Throw if block store was initialized after the consensus.
// This is needed in order to rewind consensus in case it is initialized ahead of the block store.

Reply:
deja vu 😄 - cool

Reply:
lol